lib/engines/content_block_manager/app/components/content_block_manager/content_block/document/index/date_filter_component.html.erb

```diff
@@ -34,17 +37,20 @@
     name: "last_updated_to[1i]",
     label: "Year",
     width: 4,
+    value: @filters && @filters[:last_updated_from] && @filters[:last_updated_to]["1i"],
```

Comment:
Should this be:

```suggestion
    value: @filters && @filters[:last_updated_to] && @filters[:last_updated_to]["1i"],
```

Reply:
I'm also wondering if this logic can be inside the component class itself to be a bit more DRY. You could also use [`dig`](https://apidock.com/ruby/v2_4_6/Hash/dig) Something like:

```
def date_value(date, date_part)
  filters&.dig(date, date_part)
end
```

And then call it in the view like:

```
value: date_value(:last_updated_to, "1i")
```